llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1477,6 +1477,10 @@ def TuneConditionalCompressedMoveFusion
 def HasConditionalMoveFusion : Predicate<"Subtarget->hasConditionalMoveFusion()">;
 def NoConditionalMoveFusion  : Predicate<"!Subtarget->hasConditionalMoveFusion()">;
 
+def TuneMIPSP8700
+    : SubtargetFeature<"mips-p8700", "RISCVProcFamily", "Others",
```

Comment:
Why `Others`? I think you can add a new enum value.

Reply:
I agree, thanks for suggestion: https://github.com/llvm/llvm-project/pull/120073.